pkg/remoteconfig/state/agent_config.go

```diff
@@ -26,7 +26,8 @@ type AgentConfig struct {
 
 // ConfigContent contains the configurations set by remote-config
 type ConfigContent struct {
-	LogLevel string `json:"log_level"`
+	LogLevel         string `json:"log_level"`
+	EnableStreamLogs *bool
```

Comment:
<!-- Warning represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 warning:** ‏This needs a `json:"enable_stream_logs"` to be parsed, so we can plug it with RC